### DISCUSSION TITLE
export csv can't import to postgre

### DISCUSSION THREAD

**Discussion by wonb168** [duckdb/duckdb#4760] [General]
export csv can't import to postgre,
then I find 2 different about the csv:
1. array(List) is [] exported by duckdb, but postgre need {}, eg: [1,2]
2. json(Struct) is 'exported by duckdb, but postgre need ", eg: {'a':1,'b':2}
if duckdb can add other arguments when export to csv?

**Comment by Alex-Monahan** [comment]
Could you provide a sample dataset and query that shows the issue? And could you please follow our issue template?

As a workaround, you could cast to VARCHAR and do some string manipulation prior to exporting as a csv.
